### PR TITLE
Match prototype v3

### PR DESCRIPTION
**Pull request by maysakanoni**
Update the landing, emergency, communal, postcode and address pages to match the [v3 prototype](https://southwark-repairs-alpha-master.herokuapp.com/beta/v3/).

### CODE REVIEW

File: src/components/reportRepair/address.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ const Address = ({handleChange, values, addresses}) => {
           <Link as={RouterLink} to="not-eligible">I can&apos;t find my address on this list</Link>
```
Comment (Bhavesh-Patel) [inline src/components/reportRepair/address.js:57]: The title/headline of this page is no longer "Where is the repair located?" and should be changed to "Select an address".

File: src/components/reportRepair/postcode.js [hunk 2]
```diff
@@ -28,14 +28,15 @@ const Postcode = ({handleChange, values, storeAddresses}) => {
...
-    <GridCol setWidth="two-thirds">
+    <GridCol >
       <TextInput
         value={values.postcode}
         name={'postcode'}
         onSubmit={Continue}
         validation={Validation}
         label="Postcode"
         title="What is the property address?"
+        buttonText={'Select your addresses'}
```
Comment (Bhavesh-Patel) [inline src/components/reportRepair/postcode.js:39]: The button text should be "Select your address"
```suggestion
        buttonText={'Select your address'}
```

File: src/components/reportRepair/communal.js [hunk 3]
```diff
@@ -16,9 +19,17 @@ const Communal = ({handleChange, nextStep, values}) => {
...
+      <UnorderedList>
+        <ListItem>repairs to door entry systems</ListItem>
+        <ListItem>lock repairs to communal doors</ListItem>
+        <ListItem>lighting repairs to shared areas</ListItem>
+        <ListItem>glazing repairs to shared doors or stairway windows</ListItem>
+        <ListItem>roof and gutter repairs</ListItem>
+        <ListItem>structure and external walls to your block</ListItem>
+      </UnorderedList>
     </Details>
   );
```
Comment (Bhavesh-Patel) [inline src/components/reportRepair/communal.js:35]: The "continue" button text should now be "Provide your postcode"

**Review by Bhavesh-Patel — COMMENTED** [review]
Some changes required to reflect v3 prototype

File: src/pages/landingPage.js [hunk 4]
```diff
@@ -67,31 +98,7 @@ function LandingPage() {
...
-                <ListItem>repairs to door entry systems</ListItem>
-                <ListItem>lock repairs to communal doors</ListItem>
-                <ListItem>lighting repairs to shared areas</ListItem>
-                <ListItem>glazing repairs to shared doors or stairway windows</ListItem>
-                <ListItem>roof and gutter repairs</ListItem>
-                <ListItem>structure and external walls to your block</ListItem>
-              </UnorderedList>
-            </div>
-          </Details>
+          <hr></hr>
```
Comment (Bhavesh-Patel) [inline src/pages/landingPage.js:101]: Opening hours in the prototype are different to the MVP:
Differences:

- Prototype has `Wednesday 10am - 4.30pm` and MVP has `Wednesday 9am - 4.30pm` 
- Prototype has `Thursday 9am - 5pm` and MVP has `Thursday 10am - 5pm`

File: src/components/reportRepair/emergencyRepair.js [hunk 5]
```diff
@@ -1,22 +1,45 @@
...
+        </Paragraph>
+        <UnorderedList>
+          <ListItem>Monday 9am - 5pm</ListItem>
+          <ListItem>Tuesday 9am - 5pm</ListItem>
+          <ListItem>Wednesday 10am - 4.30pm</ListItem>
+          <ListItem>Thursday 9am - 5pm</ListItem>
+          <ListItem>Friday 9am - 4.30pm</ListItem>
+        </UnorderedList>
+        <Paragraph>
+          For non-emergency requests, call **01522 873 333**
```
Comment (Bhavesh-Patel) [inline src/components/reportRepair/emergencyRepair.js:30]: Please update telephone number format to 5-6 digits, i.e. `01522 534747`

**Review by Bhavesh-Patel — CHANGES_REQUESTED** [review]
The phone number format is inconsistent in the prototype - there are both 5-3-3 and 5-6 digit formats.

UX designer has confirmed the we should consistently use 5-6 format, i.e. `01522 534747`

See specific comments for the changes needed.

Some previous comments also need addressing.

File: src/components/reportRepair/emergencyRepair.js [hunk 6]
```diff
@@ -1,22 +1,33 @@
...
+        <H3>Emergencies</H3>
+        <Paragraph>
+          An emergency is defined as something which could cause danger to
+          someone’s health or safety or cause serious damage and destruction to
+          property.
+        </Paragraph>
+        <Paragraph>
+          Emergency Out of Hours Repairs - Telephone: **01522 534747**
+        </Paragraph>
+        <Paragraph>For non-emergency requests, call **01522 873333**</Paragraph>
```
[the remark this answers is not in the document]
Comment (maysakanoni) [reply src/components/reportRepair/emergencyRepair.js]: The opening times are there, are they in the wrong place?

File: src/components/reportRepair/emergencyRepair.js [hunk 7]
```diff
@@ -1,22 +1,33 @@
...
+        <Paragraph>
+          Please do not call the emergency out of hours number if the repair is
+          not urgent. We may charge you a fee if the repair is not deemed an
+          emergency.
```
[the remark this answers is not in the document]
Comment (maysakanoni) [reply src/components/reportRepair/emergencyRepair.js]: Its still in [this](https://lincoln-repairs.herokuapp.com/beta/v3/emergency) prototype

**Pull request closed without merging**